code/controllers/Processes/playtime.dm

```diff
@@ -0,0 +1,10 @@
+/datum/controller/process/playtime/setup()
```

Comment:
why use a process for this? Just store the `world.time` once their mob has spawned, and compare that to the `world.time` if they stop playing, disconnect, or die.